src/client/components/contact-form/contact-form.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <div>Contact form</div>
```

Comment:
`Contact form` probably has some semantic value? Can it be a header, or legend for a form?

Reply:
Oeps, the `<contact-form>` component is another ticket. This shouldn't be in here.